src/helpers/distance.ts

```diff
@@ -0,0 +1,38 @@
+import { GroundPlace } from 'src/types';
+
+/**
+ * @description This function uses the ‘haversine’ formula to calculate the great-circle distance between two points.
+ *
+ * That is the shortest distance over the earth’s surface.
+ *
+ * Original calculations from 'https://www.movable-type.co.uk/scripts/latlong.html'.
+ * @param {GroundPlace} firstPlace - The first place to check, it could be a StopGroup or a StopCluster.
+ * @param {GroundPlace} secondPlace - The second place to check, it also could be a StopGroup or a StopCluster.
+ */
+const distanceBetweenTwoPlaceInKms = (firstPlace: GroundPlace, secondPlace: GroundPlace): number => {
+  const { latitude: firstPlaceLat, longitude: firstPlaceLong } = firstPlace;
+  const { latitude: secondPlaceLat, longitude: secondPlaceLong } = secondPlace;
+
+  if (!firstPlaceLat || !firstPlaceLat || !secondPlaceLat || !secondPlaceLong) {
+    throw new Error(
+      'Can\'t calculate the distance between the two places used. Please check that the values given for the "latitude" and the "longitude" are correctly setted',
+    );
+  }
+
+  // Radius of the earth in metres
+  const R = 6371e3;
+
+  // φ, λ in radians
+  const φ1 = (firstPlaceLat * Math.PI) / 180;
+  const φ2 = (secondPlaceLat * Math.PI) / 180;
+  const Δφ = ((secondPlaceLat - firstPlaceLat) * Math.PI) / 180;
```

Comment:
special character -> ☠ (@rimiti do you agree ?)

Reply:
Comment already done.